CONTRIBUTING.md

```diff
@@ -43,6 +43,14 @@ Don't forget to add your new article to the `SUMMARY.md` to let it be rendered t
 
 Please make `Draft Pull requests` early so we can follow your progress and can give early feedback (see the following section).
 
+## Style guide
+
+- Follow the book [style guide](https://github.com/rust-lang/book/blob/master/style-guide.md).
+- Follow [RFC 1574](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#appendix-a-full-conventions-text). TL;DR:
+  - Use American English
+  - Use full types name. For example `Option<T>` instead of `Option`
```

Comment:
I don't think this is necessary.

Reply:
Link them to the official documentation? Yes, it would be really cool. If that's what you mean you can suggest a change about this.

Reply:
Actually. now that I think about it we are writing types with the markdown syntax "`".
It would be silly to link to the official documentation every time.